libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.ts

```diff
@@ -316,26 +343,28 @@ export class SkyDateRangePickerComponent
 
   public ngOnInit(): void {
     if (this.#labelTextRequiredSvc) {
-      this.#labelTextRequiredSvc.validateLabelText(this.label);
+      this.#labelTextRequiredSvc.validateLabelText(
+        this.label || this.labelText,
```

Comment:
Here, and a few other spots, we need to switch the order of preference. `this.labelText || this.label`

Reply:
I would also write a unit test that confirms `labelText` is used in place of `label` when both are set.

Reply:
so the unit tests ive edited also have label set before hand and i confirm that labelText is used instead of it when set, and that it does the same for accessibility attributes.